framework/base/ErrorHandler.php

```diff
@@ -196,7 +196,7 @@ public function handleHhvmError($code, $message, $file, $line, $context, $backtr
      * @param string $message the error message.
      * @param string $file the filename that the error was raised in.
      * @param int $line the line number the error was raised at.
-     * @return bool whether the normal error handler continues.
+     * @return false whether the normal error handler continues.
```

Comment:
`@return bool whether the normal error handler continues.` is the siguature required by php, so we should not change it to `false` just because our implementation returns false only.